js/gated-content/src/views/CategoryPage.vue

```diff
@@ -3,19 +3,33 @@
     <div v-if="loading">Loading</div>
     <div v-else-if="error">Error loading</div>
     <template v-else>
-      <div class="category-details gated-container">
-        <router-link :to="{ name: 'CategoryListing' }">← Back to all categories</router-link>
-        <h2>{{ category.attributes.name }}</h2>
-        <div
-          v-if="category.attributes.description"
-          v-html="category.attributes.description.processed"
-        ></div>
+      <div class="category-details">
+        <div class="gated-container">
+          <h2>{{ category.attributes.name }}</h2>
+          <div
+            v-if="category.attributes.description"
+            v-html="category.attributes.description.processed"
+            class="mb-3"
+          ></div>
+        </div>
+      </div>
+      <div class="back-to-categories-wrapper">
+        <div class="gated-container">
+          <router-link :to="{ name: 'CategoryListing' }">← Back to all categories</router-link>
```

Comment:
Shouldn't we have a param `type` here?

Reply:
it works fine without param

Reply:
@NightHunterSV I think it reused type from the current route (that also contains this param)